Sources/GRPCHTTP2Core/Client/GRPCClientStreamHandler.swift

```diff
@@ -167,7 +179,27 @@ extension GRPCClientStreamHandler {
 
   func close(context: ChannelHandlerContext, mode: CloseMode, promise: EventLoopPromise<Void>?) {
     switch mode {
-    case .output, .all:
+    case .input:
+      context.close(mode: .input, promise: promise)
```

Comment:
Do we definitely want to forward this one? It'll close the whole stream.

Reply:
Good question. I'm not sure if we'll actually ever hit this path, but that's probably the right thing to do, yeah.